tensorflow_io/avro/20190307-avro-dataset.md

```diff
@@ -0,0 +1,299 @@
+# Avro Dataset for TensorFlow IO
+
+| Status        | Accepted                                                          |
+| :------------ | :---------------------------------------------------------------- |
+| **Author(s)** | florian.raudies@gmail.com                                         |
+| **Sponsor**   | N/A                                                               |
+| **Updated**   | 2019-03-07                                                        |
+| **Obsoletes** | N/A                                                               |
+
+## Overview
+
+Load [Avro](https://avro.apache.org/docs/current/) formatted data natively into  [TensorFlow](https://www.tensorflow.org/) from file systems that are supported by TensorFlow, e.g. the Hadoop Distributed File System ([HDFS](https://hadoop.apache.org/docs/r1.2.1/hdfs_design.html)).
```

Comment:
I suggest linking a more recent version of the docs like https://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html